app/ui/src/app/config.service.ts

```diff
@@ -62,12 +62,12 @@ export class ConfigService {
       return this.settingsRepository;
     }
 
-    if (!this.settingsRepository[group]) {
+    if (this.settingsRepository[group] === undefined) {
```

Comment:
@syndesisio/ui-api does this change looks leggit to you ? It just allows empty values, too and only croaks when no such setting provides (as the error message says anyway)

Reply:
@gashcrumb a set, empty string, shouldn't it be possible, too ? That would be much easier when using optional openshift template parameters which are empty by default. Don't really would use a 'fake' URL "false" as default value as this shows up in the OpenShift UI as 'false' in the inputfields.

Actually, it would be really awesome to allow defined, but empty values for interacting with the way how we intialize the configuration (so not only for this PR but for any other optional template parameter ending up in config.json)